sdk/resourcemanager/reservations/armreservations/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 3.0.0 (2023-03-24)
+### Breaking Changes
+
+- Field `RefundResponse` of struct `ReturnClientPostResponse` has been removed
```

Comment:
- Response of `ReturnClient.Post` has been changed from `RefundResponse` to `ReservationOrderResponse`